src/modules/authentication/components/Register/Register.tsx

```diff
@@ -1,7 +1,200 @@
+      formData.append("userName", data?.userName);
+      formData.append("email", data?.email);
+      formData.append("country", data?.country);
+      formData.append("profileImage", data?.profileImage[0]);
+      formData.append("phoneNumber", data?.phoneNumber);
+      formData.append("password", data?.password);
+      formData.append("confirmPassword", data?.confirmPassword);
+
+
+    try {
+      let response:any =await axiosInstance.post(AUTH_URL.REGISTER,formData);
+      toast.success('please verify your Registeration');
+      navigate('/verify-account');
+      // console.log(formData)
+    } catch (error:any) {
+      toast.error(error.response.data.message)
+      // console.log(error)
+    }
+  }
+  
+
   return (
-    <div>
-      <h1>Register</h1>
+    <>
+    <div className='auth-container'>
+    <div className="container-fluid bg-overlay">
+      <div className="row vh-100 justify-content-center align-items-center ">
+        <div className="col-md-8 col-lg-6   px-5 py-3 ">
+          <div className='container rounded rounded-2'>
+            <div className="logo-container  text-center">
+              <img className='w-25' src={logo} alt=''/>
+            </div>
+            <div className="title my-3">
+              <span className=''>Welcome to PMS</span>
+              <h3 className='h5'>Create New Account</h3>
+              
+            </div>
+            <form onSubmit={handleSubmit(onSubmit)} >
+            <input className='input-file' type='file'  {...register("profileImage")}/>
+              <div className='d-flex'>
+                {/* userName  */}
+              <div className='form-group px-2' >
+              <div className="input-group mb-2 input-gr">
+              <label>User Name</label>
+              <input 
+              
+              type="text"
+              className=""
+              placeholder="User Name"
+              aria-label="text"
+               aria-describedby="basic-addon1"
+              {...register('userName',{
+                required :'UserName is required',
+                
+                
+              })}
+              />
+            </div>
+            {errors.userName&&<span className='text-danger'>{errors.userName.message}</span>}
+            {/* country  */}
+            <div className="input-group mb-2 input-gr">
+            <label>Country</label>
+              <input 
+              
+              type="text"
+              className=""
+              placeholder="Country"
+              aria-label="text"
+               aria-describedby="basic-addon1"
+              {...register('country',{
+                required :'country is required'
+               
+              })}
+              />
+            </div>
+            {errors.country&&<span className='text-danger'>{errors.country.message}</span>}
+            {/* password  */}
+            <div className="input-group mb-2 input-gr">
+              <label>Password</label>
```

Comment:
We can extract that as a component and re-use in our application to avoid repeating the same logic.